src/hotspot/share/gc/g1/g1GCPhaseTimes.hpp

```diff
@@ -147,8 +147,9 @@ class G1GCPhaseTimes : public CHeapObj<mtGC> {
   };
 
   enum RestoreRetainedRegionsWorkItems {
-    RestoreRetainedRegionsFailedNum,
-    RestoreRetainedRegionsRetainedNum
+    RestoreRetainedRegionsEvacFailedNum,       // How many regions experienced an evacuation failure (pinned or allocation failure)
```

Comment:
Kind of a preexisting issue. "retained" here seems to mean evac-fail, not "kept in retain-list".